pkg/tlsclientconfig/tlsclientconfig.go

```diff
@@ -44,6 +44,10 @@ func SetupCertificates(dir string, tlsc *tls.Config) error {
 			logrus.Debugf(" crt: %s", fullPath)
 			data, err := ioutil.ReadFile(fullPath)
 			if err != nil {
+				if os.IsNotExist(err) {
+					logrus.Errorf("error reading certificate %q: %v", fullPath, err)
```

Comment:
I'm waffling between Warnf and Errorf here....  Thought I'd throw it by you for consideration.

Reply:
Since this does not fail, I believe it should be a Warnf.

Reply:
Changed it.